src/marqo/s2_inference/s2_inference.py

```diff
@@ -500,11 +517,14 @@ def _get_model_loader(model_name: str, model_properties: dict) -> Any:
     TODO: standardise these dicts
 
     Returns:
-        dict: a dictionary describing properties of the model.
+        A helper object of class `Model` (subclass depends on the type of model)
     """
 
+    # `no_model` is a special case, we use the name instead of type.
+    if model_name == SpecialModels.no_model:
+        return MODEL_PROPERTIES['loaders'][model_name]
```

Comment:
Because other models always have a `type` in their model properties. I elected not to have it for this format as it would be redundant to have any other property aside from dimensions. If we want to keep everything consistent, the user would input model properties as:
```
"model": "no_model",
"model_properties": {
   "dimensions": 123,
   "name": "no_model",
   "type": "no_model"
}
```

which I don't think is good UX.

Reply:
Instead, `no_model` just takes the `model` key and uses it for everything